packages/rn-tester/e2e/test/screenObjects/components.screen.js

```diff
@@ -0,0 +1,31 @@
+const Utils = require('../helpers/utils');
+
+class ComponentsScreen {
+
+    buttonComponent = '~Button Simple React Native button component.';
+    buttonComponentIOS = '[label="Button Simple React Native button component."]';
+
+     async checkButtonComponentIsDisplayed() {
+      let deviceLocator;
+        if (process.env.E2E_DEVICE.includes('ios')) {
+            deviceLocator = this.buttonComponentIOS;
+        }
+        if (process.env.E2E_DEVICE .includes('android')) {
+            deviceLocator = this.buttonComponent;
+        }
```

Comment:
if more of similar logic is expected, I'd suggest creating a helper like RN's [Platform.select](https://reactnative.dev/docs/platform#select), which could look like this:
```ts
const deviceLocator = platformSelect({
  ios: this.buttonComponentIOS,
  android: this.buttonComponent
});
```
Since you're using a class, it could be done once in property initializer or constructor:
```ts
class ComponentsScreen {
  element = platformSelect({
    ios: '[label="Button Simple React Native button component."]',
    android: '~Button Simple React Native button component.'
  })

  async checkButtonComponentIsDisplayed() {
    return await Utils.checkElementExistence(element);
  }
}
```